foundations/src/telemetry/log/init.rs

```diff
@@ -137,7 +137,7 @@ where
         drain,
         FieldRedactFilterFactory::new(settings.redact_keys.clone()),
     );
-    let drain = drain.filter_level(*settings.verbosity);
+    let drain = drain.filter_level((&settings.verbosity).into());
```

Comment:
We needed deref before as we were wrapping `slog::Level`. Considering that now we have `into` and verbosity is a `Copy` type, we can just have `settings.verbosity.into()` here